### ISSUE TITLE
One more metric

### ISSUE THREAD

**Issue by qumuq-til** [theojulienne/covid-19-graphics#6]
Can we have a graph that shows confirmed cases divided by daily tests ratio? It could be useful.

**Comment by theojulienne** [comment]
Is this the % case positivity? If so, for countries where we have test data available, the tests graph shows % case positivity as well - for example, in the US:

<img width="1164" alt="Screen Shot 2020-11-10 at 8 59 17 am" src="https://user-images.githubusercontent.com/349190/98601354-06d72780-2333-11eb-8063-fe09a3498e25.png">

This is a good indicator of whether testing is adequate for the level of transmission. A lot of the countries on the site don't have testing data, back when the data feeds were first set up only the US data site had them - but perhaps now it's easier to find elsewhere as well.

**Comment by qumuq-til** [comment]
Yeah, it's that thing. I missed it probably because as you said the data itself is missing for many countries...
